h/views/api_auth.py

```diff
@@ -20,6 +20,7 @@ def access_token(request):
         'access_token': token.value,
         'token_type': 'bearer',
         'expires_in': TOKEN_TTL.total_seconds(),
+        'refresh_token': token.refresh_token,
```

Comment:
Hmm, all I could find was this: https://tools.ietf.org/html/rfc6749#section-4.2.2

`expires_in` is RECOMMENDED, and if it's left out then we SHOULD provide the expiration time "by other means" or document its default value. The spec seems like it might be written to assume that all tokens expire.

My reading of that, though, is that we could omit `expires_in` for non-expiring tokens and then document, in our API docs, that when there's no `expires_in` the default is that the token doesn't expire.

So as far as the code goes, we can just change it to omit `expires_in` from the response if the token has no expires.